### ISSUE TITLE
Show a warning when opening another website than getnightingale.com

### ISSUE THREAD

**Issue by darealshinji** [nightingale-media-player/nightingale-hacking#362]
Since the xulrunner engine is very outdated it might be a good idea to show a warning when the user is going to visit another site than getnightingale.com. Ideally this warning will only be displayed once or it has an option to never show up again.
##

**Comment by freaktechnik** [comment]
I don't think this is the roght approach - but I agree with the idea. Reasons why we shouldn't do this os becayse add-ons open tabs too, and I don't think having a specific listof allowed pages is the solution to how the browser part currently works. There are also many other places tha tabs that make nightingale potentially vulnerable btw. like the youtube iFrame in mashtape.
